fc_util/src/lib.rs

```diff
@@ -3,57 +3,189 @@
 
 extern crate libc;
 
+use std::fmt;
+
 pub mod validators;
 
-pub fn timestamp_cycles() -> u64 {
-    #[cfg(target_arch = "x86_64")]
-    // Safe because there's nothing that can go wrong with this call.
-    unsafe {
-        std::arch::x86_64::_rdtsc() as u64
+/// Constant to convert seconds to nanoseconds.
+pub const SECONDS_TO_NANOS: u64 = 1_000_000_000;
```

Comment:
`SECONDS_TO_NANOS` is not the best name for that constant. Can you change it to something that makes more sense?

Examples:
- `SEC_TO_NANO_MULTIPLIER`
- `SECOND_TO_NANOSECOND`
- `SEC_TO_NANO`

Basically I would name it so that it's clear that it's a constant which represents the multiplier for converting one second to one nanosecond.

Better yet, you can create a public function: `seconds_to_nanoseconds` which would farther improve the readability of the code.

```
// In fc_util:
pub fn seconds_to_nanoseconds(value: i64) -> Option<i64> {
        value.checked_mul(1_000_000_000)
}

// Code that was using the constant would look like this:
self.tick_offset = seconds_to_nanoseconds(i64::from(val)).unwrap();
// In some cases you might want to check the value of seconds_to_nanoseconds. 
// In RTC if that value overflows we most likely have a malicious user and we want to terminate
// the execution of Firecracker so we use unwrap.

```

Reply:
`NANOS_PER_SECOND` 🐎